integration-cli/docker_cli_history_test.go

```diff
@@ -67,16 +66,14 @@ RUN echo "Z"`,
 }
 
 func (s *DockerSuite) TestHistoryExistentImage(c *check.C) {
-	historyCmd := exec.Command(dockerBinary, "history", "busybox")
-	_, exitCode, err := runCommandWithOutput(historyCmd)
+	_, exitCode, err := dockerCmdWithError(c, "history", "busybox")
 	if err != nil || exitCode != 0 {
 		c.Fatal("failed to get image history")
 	}
 }
 
 func (s *DockerSuite) TestHistoryNonExistentImage(c *check.C) {
-	historyCmd := exec.Command(dockerBinary, "history", "testHistoryNonExistentImage")
-	_, exitCode, err := runCommandWithOutput(historyCmd)
+	_, exitCode, err := dockerCmdWithError(c, "history", "testHistoryNonExistentImage")
```

Comment:
We are expecting command to be failed, so I think here need a `dockerCmdWithError`.